package.json

```diff
@@ -23,25 +23,25 @@
   },
   "devDependencies": {
     "babel-eslint": "^8.2.3",
-    "create-universal-package": "3.4.4",
-    "eslint": "4.19.1",
+    "create-universal-package": "^3.4.4",
+    "eslint": "^4.19.1",
     "eslint-config-fusion": "^1.0.1",
-    "eslint-plugin-cup": "1.0.2",
-    "eslint-plugin-flowtype": "^2.46.2",
+    "eslint-plugin-cup": "^1.0.2",
+    "eslint-plugin-flowtype": "^2.46.3",
     "eslint-plugin-import": "^2.11.0",
-    "eslint-plugin-prettier": "2.6.0",
-    "eslint-plugin-react": "7.7.0",
+    "eslint-plugin-prettier": "^2.6.0",
+    "eslint-plugin-react": "^7.8.2",
     "flow-bin": "^0.72.0",
-    "fusion-core": "^1.2.3",
+    "fusion-core": "^1.3.0",
     "fusion-react-async": "^1.2.2",
     "fusion-test-utils": "^1.0.5",
-    "nyc": "^11.6.0",
+    "nyc": "^11.8.0",
     "prettier": "^1.12.1",
     "prop-types": "^15.6.1",
     "react": "^16.3.2",
     "react-dom": "^16.3.2",
-    "tape-cup": "4.7.1",
-    "unitest": "2.1.1"
+    "tape-cup": "^4.7.1",
+    "unitest": "^1.1.1"
```

Comment:
Looks like these deps got downgraded somehow?

Reply:
[v1.1.1 appears to be the latest published version](https://www.npmjs.com/package/unitest).

Reply:
@rtsao  - Something we should probably look into.